src/cm1.F

```diff
@@ -2259,15 +2259,18 @@ program cm1
       allocate( reqt(d3t) )
       reqt = 0
 
+      if ( nparcelsInit .gt. nparcelsMax ) then
+          if ( myid .eq. 0 ) print *,' WARNING: nparcelsInit exceeds allowed max. Check nparcelsMax. Exiting...'
```

Comment:
@sjsprecious  I am concerned that you have removed the necessary padding for setting the value for nparcelsLocal.  How do you prevent code failure if the number of incoming particles exceeds nparcelsLocal.

Reply:
@johnmauff with the new namelist variable `nparcelsMax`, `nparcelsLocal` is now calculated as `nparcelsMax/numproc` and I think we do not need the `padding` variable now. In the `init3d.F` subroutine, there is a check if the number of incoming particles exceeds `nparcelsLocal` and an error will issue accordingly.

Reply:
@sjsprecious I think the padding still needs to be present.  Right now the code could fail unpredictably later in the simulation if the number of incoming droplets crosses is sufficiently large.

Reply:
@johnmauff thanks for the detailed explanation. That makes sense. However, if we use the variable `padding` to increase the size of `nparcelsLocal`, how could we ensure that the total droplets after injection will not exceed `nparcelsMax` at each time step?